storage/innobase/xtrabackup/doc/source/howtos/recipes_ibkx_gtid.rst

```diff
@@ -54,7 +54,7 @@ STEP 4: Configure and start replication
 
 Following command will tell the new slave what was the last ``GTID`` executed on the master when backup was taken.
```

Comment:
examples need to use xtrabackup:
```
xtrabackup --backup --target-dir=data/backups/
```
for backup, and 

```
xtrabackup --prepare --target-dir=/data/backups
```
for prepare

also output will be
```
xtrabackup: MySQL binlog position: filename 'mysql-bin.000002', position 1232, GTID of the last change 'c777888a-b6df-11e2-a604-080027635ef5:1-4'
```